server/src/storage/segment.rs

```diff
@@ -165,15 +228,18 @@ mod tests {
     }
```

Comment:
Is this test missing an assert? If not a comment should probably be added with information about what the test checks.

Reply:
I am not quite sure what you mean, the tests checks that the `is_full` function returns `true` when the segment is full.

Reply:
The comment was because I misunderstood `test_check_segment_is_full()` due to GitHub's interface, it implementation makes sense.